packages/nextjs/components/scaffold-eth/Balance.tsx

```diff
@@ -39,7 +41,7 @@ export const Balance = ({ address, className = "" }: TBalanceProps) => {
         {isEthBalance ? (
```

Comment:
~~Should we can rename this variable as well to something more fitting~~

On reading the changes further, I think we'd have to rename too many things... Even `scaffold-eth` technically mentions eth... So let's not go that route :sweat_smile:

Reply:
Are you suggesting undoing all the renaming already done?